backend/src/transaction/transaction.controller.ts

```diff
@@ -324,4 +324,36 @@ export default class TransactionController {
       throw error;
     }
   }
+
+  @Get('/:transactionId/detail')
```

Comment:
@M-Francia it coould just be "transactionId", no need to add "detail"

Reply:
we can't as we have   @Get(':publicKey')